code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Dispenser.dm

```diff
@@ -16,6 +16,18 @@
 	reagent_state = SOLID
 	color = "#e9e6e4"
 
+//VOREStation Edit
+/datum/reagent/calcium/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
+	if(ishuman(M) && rand(1,1000) == 1) // Higher chance of occuring than with milk, since most people don't just eat straight calcium
```

Comment:
I would put this at 1,10000, because I know some people are going to powergame this and start chugging straight calcium if they do not have access to osteodaxon.